NaToxAq/NA000208.txt

```diff
@@ -4,12 +4,11 @@ DATE: 2019.02.28
 AUTHORS: Tobias Schulze, Jawameer Hama, Hubert Schupke, Martin Krauss, Helmholtz Centre for Environmental Research GmbH - UFZ, Leipzig, Germany and University of Copenhagen (UCPH), Denmark
 LICENSE: CC BY
 COPYRIGHT: Copyright (C) 2019
-PUBLICATION: 
 COMMENT: CONFIDENCE Reference Standard (Level 1)
 COMMENT: INTERNAL_ID 3
 CH$NAME: Angustifoline
 CH$NAME: VTIPIBIDDZPDAV-IFWUJCSASA-N
```

Comment:
There are also InChIKeys as names ... the infolist should have been checked before processing!

Reply:
I have to think about a way to treat this in the data repo... I'll put it on the wish list.